exchange_info_service_test.go

```diff
@@ -49,7 +49,7 @@ func (s *exchangeInfoServiceTestSuite) TestExchangeInfo() {
 				QuotePrecision:     8,
 				OrderTypes:         []string{"LIMIT", "LIMIT_MAKER", "MARKET", "STOP_LOSS_LIMIT", "TAKE_PROFIT_LIMIT"},
 				IcebergAllowed:     true,
-				Filters: []map[string]string{
+				Filters: []map[string]interface{}{
```

Comment:
hi @ethanchewy , thanks for the update. 
Would you pls add more filters with integer value to the test case as in https://github.com/binance-exchange/binance-official-api-docs/blob/master/rest-api.md#symbol-filters, so that we can cover this change in unit test, thanks.